orchestrator/orchestrator/src/main_loop.rs

```diff
@@ -408,14 +456,3 @@ pub async fn eth_signer_main_loop(
         );
     }
 }
-
-pub async fn check_for_eth(orchestrator_address: EthAddress, eth_client: EthClient) {
```

Comment:
There was an existing issue around the orchestrator not functioning correctly if a tiny amount of eth had not been deposited into the delegate key wallet. Did that stem from this error, or is it a problem regardless and that's why this check existed?

Reply:
I think I removed this because it's a duplicate. There's another copy, the one that's actually used, in gravity_utils/connection_prep.rs

Reply:
Is it an issue with the address not existing yet if it has no funds?

Reply:
I think I removed this because it's a duplicate. There's another copy, the one that's actually used, in gravity_utils/connection_prep.rs

The only place not in gorc that it's used is when running the relayer